content/300-guides/050-database/300-seed-database.mdx

```diff
@@ -185,18 +150,24 @@ To seed your database with TypeScript:
 5. To seed the database, run the `db seed` CLI command:
 
    ```
-   prisma db seed --preview-feature
+   npx prisma db seed
    ```
 
-## Seeding your database with JavaScript
+ 
+</tab> 
 
-To seed your database with JavaScript:
+ 
+<tab> 
 
+ 
 1. Create a new file named `seed.js` in the same location as your `schema.prisma`.
-2. In the `seed.js` file, import Prisma Client, initialize it and create some records. The following example uses a script, but you can also [use an `export`](#requirements-for-seeding-with-typescript-or-javascript):
+2. In the `seed.js` file, import Prisma Client, initialize it and create some records.
+
+  <details><summary>Expand to view example seed script</summary> 
 
+  
    ```js
-   const { PrismaClient } = require('@prisma/client')
+   import { PrismaClient } from '@prisma/client'
```

Comment:
If its a regular JS project then
```
const { PrismaClient } = require('@prisma/client')
```
only will work by default

Reply:
Hmm i guess it depends how the project is setup. If the user has `"types": "module"` in their `package.json` then import syntax is valid. Maybe im just being pedantic about this though and should change it to require instead

Reply:
I changed it back to require.

Reply:
Yes it definitely depends on the project config